test/configtest/configtest.test.js

```diff
@@ -41,7 +42,7 @@ describe('basic info usage', () => {
         const { exitCode, stderr, stdout } = run(__dirname, ['configtest', './a.js'], false);
 
         expect(exitCode).toBe(2);
-        expect(stderr).toContain(`The specified config file doesn't exist`);
+        expect(stderr).toContain(`The specified config file doesn't exist in '${path.join(__dirname, './a.js')}`);
```

Comment:
This change doesn't add any value to test IMO.
The aim for the test is to look for error when configuration doesn't exists, which previous one was also doing. Unnecessary regressions must be avoided as they lead to unstability.